src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingHome.java

```diff
@@ -45,6 +46,27 @@
     @NotNull
     private final List<BlockPos> bedList = new ArrayList<>();
 
+    /**
+     * Is a female citizen living here?
+     */
+    private boolean femalePresent = false;
+
+    /**
+     * Is a male citizen living here?
+     */
+    private boolean malePresent = false;
```

Comment:
can't we make this a method and calculate it?

Reply:
we could, but I kinda like two booleans instead of regular recalculations more